charts/shoot-core/charts/calico/templates/calico.yaml

```diff
@@ -454,3 +454,26 @@ spec:
     kind: NetworkPolicy
     plural: networkpolicies
     singular: networkpolicy
+
+---
+
+apiVersion: crd.projectcalico.org/v1
+kind: GlobalNetworkPolicy
+metadata:
+  name: block-metadata-service
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  selector: all()
+  ingress:
+  - action: Allow
+  egress:
+  - action: Allow
```

Comment:
Shouldn't the action be Deny with nets instead of Allow with notNets?

Reply:
You need `Allow` otherwise no traffic can be send / received

Reply:
Ah. Because of the default Deny?

Reply:
```yaml
apiVersion: crd.projectcalico.org/v1
kind: GlobalNetworkPolicy
metadata:
  name: block-metadata-service
  labels:
spec:
  egress:
  - action: Deny
    destination:
      nets:
      - 169.254.169.254/32
  types:
  - Egress
```

blocks everything